src/wasm-lib/kcl/src/errors.rs

```diff
@@ -295,7 +295,6 @@ pub struct CompilationError {
 }
 
 impl CompilationError {
-    #[allow(dead_code)]
```

Comment:
I usually use `#[expect(dead_code)]` so that the exception gets removed when it's no longer needed.